net/DevExtreme.AspNet.Data.Tests.EFCore/SampleLoadOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DevExtreme.AspNet.Data.Tests.EFCore {
+
+    class SampleLoadOptions : DataSourceLoadOptionsBase {
```

Comment:
Yes, but why is it abstract?
Just to have a excuse to create such strange-looking classes?